pkg/collectinfo/collectinfo_util.go

```diff
@@ -206,6 +230,103 @@ func captureObject(logger *zap.Logger, k8sClient client.Client, gvk schema.Group
 	return nil
 }
 
+func captureSummary(logger *zap.Logger, ns, rootOutputPath string) error {
+	cmdMap := make(map[string]*exec.Cmd)
```

Comment:
I think we should check if `kubectl` binary is available in the PATH or not before trying to create summary.
Also, if it's not available in the PATH, should we ask for `kubectl` path as param or make it pre-req in our doc ?